services/common/src/components/forms/ViewForm.spec.tsx

```diff
@@ -0,0 +1,309 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { ReduxWrapper } from "@mds/common/tests/utils/ReduxWrapper";
+import FormWrapper from "./FormWrapper";
+import { Field, FieldArray, FormSection } from "./form";
+import RenderAutoComplete from "./RenderAutoComplete";
+import RenderAutoSizeField from "./RenderAutoSizeField";
+import RenderCancelButton from "./RenderCancelButton";
+import RenderSubmitButton from "./RenderSubmitButton";
+import RenderCheckbox from "./RenderCheckbox";
+import RenderDate from "./RenderDate";
+import RenderDateTimeTz from "./RenderDateTimeTz";
+import RenderField from "./RenderField";
+import RenderFileUpload from "./RenderFileUpload";
+import RenderGroupCheckbox from "./RenderGroupCheckbox";
+import RenderGroupedSelect from "./RenderGroupedSelect";
+import RenderHiddenField from "./RenderHiddenField";
+import RenderLargeSelect from "./RenderLargeSelect";
+import RenderMultiSelect from "./RenderMultiSelect";
+import RenderOrgBookSearch from "./RenderOrgBookSearch";
+import RenderRadioButtons from "./RenderRadioButtons";
+import RenderResetButton from "./RenderResetButton";
+import RenderRichTextEditor from "./RenderRichTextEditor";
+import RenderSelect from "./RenderSelect";
+import RenderTime from "./RenderTime";
+import RenderTreeSelect from "./RenderTreeSelect";
+import { normalizeDatetime } from "@mds/common/redux/utils/helpers";
+
+const initialValues = {
```

Comment:
the big test!
Put in all the input components. Did some field nesting. Make sure something sensible is returned (nothing crashes, values show up). Couple things I decided not to worry about because nowhere in the system does it matter (RenderCascader, Fields). If it becomes necessary in the future we'll deal with it then.